napari/components/viewer_model.py

```diff
@@ -1370,9 +1381,11 @@ def _add_layers_with_plugins(
                 paths, plugin=plugin, stack=stack
             )
 
+        if layer_data is None:
+            return []
```

Comment:
Why this addition?

Reply:
My guess is typing fails 3 line below:

```
len(paths) == len(layer_data)
```

`len(None)` make no sens.

Reply:
Ah yeah, got it :+1: